atomicapp/providers/kubernetes.py

```diff
@@ -203,4 +203,8 @@ def undeploy(self):
             cmd = [self.kubectl, "delete", "-f", path, "--namespace=%s" % self.namespace]
```

Comment:
what happens in `_scale_replicas()` above if the replica doesn't exist? Probably need to ignore_errors there too. 

actually why don't we have a instance variable `self.ignore_errors` that we can set before we even call `undeploy()` and then `_call()` can check it. 

So in nulecule/base.py we would call `provider.ignore_errors = True` and then `provider.undeploy()` - or something like that.